sup3r/preprocessing/data_handling.py

```diff
@@ -623,8 +624,11 @@ def time_chunk_size(self):
         if self._time_chunk_size is None:
             step_mem = self.feature_mem * len(self.raw_features)
             step_mem /= len(self.time_index)
-            self._time_chunk_size = np.min([np.int(1e9 / step_mem),
-                                            self.n_tsteps])
+            if step_mem == 0:
```

Comment:
this should probably just be n_tsteps, although it won't really matter.

Reply:
step_mem -> n_tsteps?

Reply:
or you're saying i should set `self._time_chunk_size = self.n_tsteps`?

Reply:
Yeah time_chunk_size = n_steps. It'll cap to that anyway with 1e9.